src/views/components/course_list.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+
+<div>
+    <h1>Current Courses</h1>
+    <div v-for="course in current_courses">
+      {{course}}
+    </div>
+
+    <h1>Past Courses</h1>
+    <div v-for="course in past_courses">
+      {{course}}
+    </div>
+</div>
+
+</template>
+
+<script>
+import get_unique_attribute from "./get_unique_attribute";
+import generate_query_string from "./generate_query_string";
+export default {
+    name: "CourseList",
+    props: ["term"],
+    data: function() {
+        return {
+            current_courses: null,
+            past_courses: null
+        };
+    },
+    created() {
+        let url = {
+            what: "course_pairings",
+            user_id: this.$route.params.user_id,
+            column: "instructor"
+        };
+        fetch("get_info.php?" + generate_query_string(url))
+            .then(res => res.json())
+            .then(data => {
+                let current_courses = data.DATA.filter(

```

Comment:
I've seen `DATA[0]` in a lot of places. Are you sure it's just `DATA` here?

Reply:
Yes, `DATA` here gives an array of objects that I'm using to filter through and get what I need. `DATA[0]` being used else where was that the API for surveys always return a list of surveys(even if it's only), thus when I use a component to call and render data for only 1 survey, I still needed to index into the array to get the survey data.